source/refined-github.css

```diff
@@ -177,3 +177,22 @@ pr-branches
 	min-height: 4em;
 	height: 4em;
 }
+
+/* Improve wrapping of repo header buttons on smaller screens #5398 */
+@media (min-width: 768px) {
+	#repository-container-header > :first-child > :first-child {
+		flex-shrink: 0 !important;
+	}
+
+	.pagehead-actions {
+		display: flex !important;
+		flex-shrink: 1 !important;
+		flex-wrap: wrap !important;
+		justify-content: flex-end;
+		gap: 8px;
+	}
+
+	.pagehead-actions li {
+		margin-right: 0 !important;
+	}
```

Comment:
Needed because this is replaced by `gap` on the parent.

Reply:
I'm not sure why but I'm seeing this on Chrome

<img width="488" alt="Screen Shot" src="https://user-images.githubusercontent.com/1402241/167236300-b6209f61-4567-4b03-a3dd-f0fd46ee3cdb.png">

Reply:
That's likely because the `display: flex` didn't apply, the selector just needs to be made more specific.